### PR TITLE
Updated readme github buttons

### PR DESCRIPTION
**Pull request by swathiswaminathan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,20 @@ In the diagram, note:
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```
Comment (emailbob) [inline README.md]: `[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)`

**Comment by emailbob** [comment]
This project already has a Google Cloud Shell button. We need to keep one of them.

File: README.md (hunk 1 above)
Comment (swathiswaminathan) [reply README.md]: please test this

**Comment by emailbob** [comment]
test this please

**Pull request merged**